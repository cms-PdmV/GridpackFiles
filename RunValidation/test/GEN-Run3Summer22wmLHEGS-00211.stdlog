000 (8360713.000.000) 06/03 10:09:31 Job submitted from host: <188.184.90.62:9618?addrs=188.184.90.62-9618+[2001-1458-201-e4--100-238]-9618&alias=bigbird16.cern.ch&noUDP&sock=schedd_3504972_d6e3>
...
040 (8360713.000.000) 06/03 10:11:10 Started transferring input files
	Transferring to host: <188.184.160.10:9618?addrs=188.184.160.10-9618+[2001-1458-301-48--100-4]-9618&alias=b7s09p9183.cern.ch&noUDP&sock=slot1_8_123958_5fea_83330>
...
040 (8360713.000.000) 06/03 10:11:10 Finished transferring input files
...
001 (8360713.000.000) 06/03 10:11:11 Job executing on host: <188.184.160.10:9618?addrs=188.184.160.10-9618+[2001-1458-301-48--100-4]-9618&alias=b7s09p9183.cern.ch&noUDP&sock=startd_28549_d160>
...
006 (8360713.000.000) 06/03 10:11:30 Image size of job updated: 1989764
	24  -  MemoryUsage of job (MB)
	24276  -  ResidentSetSize of job (KB)
...
006 (8360713.000.000) 06/03 10:16:31 Image size of job updated: 2799804
	168  -  MemoryUsage of job (MB)
	171324  -  ResidentSetSize of job (KB)
...
006 (8360713.000.000) 06/03 10:21:31 Image size of job updated: 3468640
	320  -  MemoryUsage of job (MB)
	327436  -  ResidentSetSize of job (KB)
...
006 (8360713.000.000) 06/03 10:26:32 Image size of job updated: 3468640
	353  -  MemoryUsage of job (MB)
	360584  -  ResidentSetSize of job (KB)
...
006 (8360713.000.000) 06/03 10:31:32 Image size of job updated: 3682144
	462  -  MemoryUsage of job (MB)
	472840  -  ResidentSetSize of job (KB)
...
006 (8360713.000.000) 06/03 10:36:32 Image size of job updated: 4066672
	665  -  MemoryUsage of job (MB)
	680636  -  ResidentSetSize of job (KB)
...
006 (8360713.000.000) 06/03 10:41:33 Image size of job updated: 7097992
	3723  -  MemoryUsage of job (MB)
	3811716  -  ResidentSetSize of job (KB)
...
040 (8360713.000.000) 06/03 10:41:56 Started transferring output files
...
040 (8360713.000.000) 06/03 10:41:56 Finished transferring output files
...
005 (8360713.000.000) 06/03 10:41:56 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:03:20, Sys 0 00:00:15  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:03:20, Sys 0 00:00:15  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	44189  -  Run Bytes Sent By Job
	915  -  Run Bytes Received By Job
	44189  -  Total Bytes Sent By Job
	915  -  Total Bytes Received By Job
	Partitionable Resources :     Usage  Request Allocated 
	   Cpus                 :      0.03        4         4 
	   Disk (KB)            : 281761           1   2991781 
	   Memory (MB)          :   3723        8192      8200 

	Job terminated of its own accord at 2023-06-03T08:41:56Z.
...
