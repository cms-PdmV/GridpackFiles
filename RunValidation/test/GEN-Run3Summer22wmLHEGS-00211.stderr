WARNING: Overriding HOME environment variable with APPTAINERENV_HOME is not permitted
WARNING: In non-interactive mode release checks e.g. deprecated releases, production architectures are disabled.
WARNING: In non-interactive mode release checks e.g. deprecated releases, production architectures are disabled.
--2023-06-03 10:12:11--  https://cms-pdmv.cern.ch/mcm/public/restapi/requests/get_fragment/GEN-Run3Summer22wmLHEGS-00211/0
Resolving cms-pdmv.cern.ch (cms-pdmv.cern.ch)... 188.185.84.85
Connecting to cms-pdmv.cern.ch (cms-pdmv.cern.ch)|188.185.84.85|:443... connected.
WARNING: The certificate of 'cms-pdmv.cern.ch' is not trusted.
WARNING: The certificate of 'cms-pdmv.cern.ch' hasn't got a known issuer.
HTTP request sent, awaiting response... 200 OK
Length: 1932 (1.9K) [text/plain]
Saving to: 'GEN-Run3Summer22wmLHEGS-00211.py'

     0K .                                                     100% 15.9M=0s

2023-06-03 10:12:12 (15.9 MB/s) - 'GEN-Run3Summer22wmLHEGS-00211.py' saved [1932/1932]

WARNING: In non-interactive mode release checks e.g. deprecated releases, production architectures are disabled.
ERROR: Can not setup your current working area for SCRAM_ARCH: $ENV{SCRAM_ARCH}
Your current development area ${location}/${areaname}
is using a different ${areaname}/config then the one used for
/cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_4_8
./runcmsgrid.sh: line 41: cd: CMSSW_12_4_8/src: No such file or directory
03-Jun-2023 10:41:41 CEST  Initiating request to open LHE file thread0/cmsgrid_final.lhe
03-Jun-2023 10:41:41 CEST  Successfully opened LHE file thread0/cmsgrid_final.lhe
03-Jun-2023 10:41:41 CEST  Initiating request to open LHE file thread0/cmsgrid_final.lhe
03-Jun-2023 10:41:41 CEST  Successfully opened LHE file thread0/cmsgrid_final.lhe
%MSG-w LogicError:  Pythia8ConcurrentHadronizerFilter:generator@streamBeginRun  03-Jun-2023 10:41:48 CEST Run: 1 Stream: 0
::getByLabel: An attempt was made to read a Run product before endRun() was called.
The product is of type 'LHERunInfoProduct'.
The specified ModuleLabel was 'externalLHEProducer'.
The specified productInstanceName was ''.

%MSG
Begin processing the 1st record. Run 1, Event 1, LumiSection 1 on stream 0 at 03-Jun-2023 10:41:53.433 CEST
03-Jun-2023 10:41:54 CEST  Closed LHE file thread0/cmsgrid_final.lhe

------------------------------------
GenXsecAnalyzer:
------------------------------------
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
Overall cross-section summary 
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Process		xsec_before [pb]		passed	nposw	nnegw	tried	nposw	nnegw 	xsec_match [pb]			accepted [%]	 event_eff [%]
0		1.356e+07 +/- 5.173e+05		0	0	0	1	1	0	0.000e+00 +/- 0.000e+00		0.0 +/- 0.0	0.0 +/- 0.0
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
Total		1.356e+07 +/- 5.173e+05		0	0	0	1	1	0	0.000e+00 +/- 0.000e+00		0.0 +/- 0.0	0.0 +/- 0.0
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Before matching: total cross section = 1.356e+07 +- 5.173e+05 pb
After matching: total cross section = 0.000e+00 +- 0.000e+00 pb
Matching efficiency = 0.0 +/- 0.0   [TO BE USED IN MCM]

After filter: final cross section = 0.000e+00 +- 0.000e+00 pb
After filter: final fraction of events with negative weights = 0.000e+00 +- 0.000e+00
After filter: final equivalent lumi for 1M events (1/fb) = 0.000e+00 +- 0.000e+00
TimeReport> Time report complete in 1643.57 seconds
 Time Summary: 
 - Min event:   0.97126
 - Max event:   0.97126
 - Avg event:   0.97126
 - Total loop:  1497.63
 - Total init:  145.916
 - Total job:   1643.57
 - EventSetup Lock: 0
 - EventSetup Get:  0
 Event Throughput: 0.000667723 ev/s
 CPU Summary: 
 - Total loop:     224.814
 - Total init:     19.3162
 - Total extra:    0
 - Total children: 76.4561
 - Total job:      244.16
 Processing Summary: 
 - Number of Events:  1
 - Number of Global Begin Lumi Calls:  1
 - Number of Global Begin Run Calls: 1


=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 GenXSecAnalyzer      -w GenXSecAnalyzer:                      13       13
    2 LogicError           -w Pythia8Concurren                       1        1
    3 TimeReport           -e AfterModEndJob                         1        1
    4 fileAction           -s ExternalLHEProdu                       4        4
    5 fileAction           -s ExternalLHEProdu                       1        1

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 GenXSecAnalyzer      EndJob           EndJob           EndJob
    2 LogicError           Run: 1 Stream: 0                  
    3 TimeReport           EndJob                            
    4 fileAction           Run: 1           Run: 1           Run: 1
    5 fileAction           End Run: 1                        

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
Warning                14                  14
Error                   1                   1
System                  5                   5

dropped waiting message count 0
